### PR TITLE
Standardize: global-shortcuts, ipc (main), ipc (render)

### PR DESCRIPTION
**Pull request by jlord**
Updating the docs per the [docs style guide](https://github.com/atom/electron/blob/master/docs/styleguide.md). 

Definitely would like @zcbenz :eyes: here. I've made quite a few edits to the `ipc` docs and want to make sure they're :ok:

### CODE REVIEW

File: docs/api/global-shortcut.md [hunk 1]
```diff
@@ -40,18 +47,18 @@ app.on('will-quit', function() {
...
 
 Returns `true` or `false` depending on whether the shortcut `accelerator` is registered.
 
-## globalShortcut.unregister(accelerator)
+### `globalShortcut.unregister(accelerator)`
 
 * `accelerator` [Accelerator](accelerator.md)
 
-Unregisters the global shortcut of `keycode`.
+Unregisters the global shortcut of `accelerator`.
```
Comment (jlord) [inline docs/api/global-shortcut.md]: cc @zcbenz — I want to double check that using `accelerator` here is correct.

File: docs/api/ipc-renderer.md [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
-See [ipc (main process)](ipc-main-process.md) for examples.
+See [ipc (main process)](ipc-main-process.md) for code examples.
+
+## Methods
+
+The `ipc` module has the following methods for sending messages:
+
+**Note**: When using these methods to send a `message` you must also listen for it in the main process with [`ipc (main process)`](ipc-main-process.md).
+
+### `ipc.send(channel[, message])`
```
Comment (zcbenz) [inline docs/api/ipc-renderer.md]: Actually `ipc.send` can send arbitrary numbers of arguments, like `ipc.send('channel', 1, 2, 3, 4, 5)`, which behaves the same with `EventEmitter.emit`, maybe we can follow [Node's conventions](https://nodejs.org/api/events.html#events_emitter_emit_event_arg1_arg2) here by using `(channel[, arg1][, arg2][, ...])`.

File: docs/api/global-shortcut.md (hunk 1 above)
Comment (zcbenz) [inline docs/api/global-shortcut.md]: Definitely should be `accelerator` here.

File: docs/api/ipc-renderer.md (hunk 2 above)
Comment (jlord) [inline docs/api/ipc-renderer.md]: Ahh, gotcha :+1:

**Comment by jlord** [comment]
This one is ready! 

I'd like to double check @zcbenz that how wrote this method and described it looks good (repeated it a few times exchanging async for sync and such):

---

### `ipc.sendSync(channel[, arg1][, arg2][, ...])`

* `channel` String - The event name.
* `message` (optional)

Send an event to the main process synchronously via a `channel`. Optionally,
there can be a message: one or a series of arguments, `arg`, which can have any
type. The main process handles it by listening for the `channel` event with `ipc`.

**Comment by zcbenz** [comment]
Looks good to me :+1:.

**Comment by jlord** [comment]
:sparkles:

**Pull request merged**